PhysicsTools/PatUtils/interface/MiniIsolation.h

```diff
@@ -8,19 +8,18 @@
   of packed PF candidates
 
   Mini-Isolation reference: https://hypernews.cern.ch/HyperNews/CMS/get/susy/1991.html
-
+    typedef ROOT::Math::LorentzVector<ROOT::Math::PxPyPzE4D<double> > LorentzVector;
```

Comment:
It's not, that's a mistake.. must have accidentally pasted it after I deleted it below and didn't catch it. I can remove it